source/includes/read/retrieve.rb

```diff
@@ -0,0 +1,47 @@
+require 'bundler/inline'
+
+gemfile do
+  source 'https://rubygems.org'
+  gem 'mongo'
+end
+
+uri = '<connection string>'
+
+begin
+  client = Mongo::Client.new(uri)
+
+  # start-db-coll
+  database = client.use('sample_training')
+  collection = database[:companies]
+  # end-db-coll
+
+  # Finds one document with a "name" value of "LinkedIn"
+  # start-find-one
+  document = collection.find({ 'name' => 'LinkedIn' }).first
```

Comment:
This isn't wrong, but it might be more idiomatically written as:

```suggestion
  document = collection.find(name: 'LinkedIn').first
```